test/e2e/specs/publishing.test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import '../support/bootstrap';
+import { newPost, newDesktopBrowserPage, wait } from '../support/utils';
+
+describe( 'Publishing', () => {
+	beforeAll( async () => {
+		await newDesktopBrowserPage();
+	} );
+
+	beforeEach( async () => {
+		await newPost();
+	} );
+
+	it( 'Should publish a post and close the panel once we start editing again', async () => {
+		await page.type( '.editor-post-title__input', 'E2E Test Post' );
+
+		// Opens the publish panel
+		await page.click( '.editor-post-publish-panel__toggle' );
+
+		// Wait for a second ( wait for the animation )
+		await wait( 1000 );
```

Comment:
Is there a way to express this with `waitForSelector` (e.g. a class on the available button)? How else can we discourage using `wait` as a crutch? Could we call this something like `waitDurationYesImCertainWaitForSelectorIsNotABetterOption( 1000 )` ?